app/src/main/java/com/soen490chrysalis/papilio/create_activity.kt

```diff
@@ -0,0 +1,32 @@
+package com.soen490chrysalis.papilio
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.view.MenuItem
+
+class create_activity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_create_activity)
+
+        // Create Action Bar var so we can 1) display it with a proper title and 2) put a working back button on it
+        var actionBar = supportActionBar
+
+        // if Action Bar is not null, then put a back button on it as well as put the "User Profile" title on it
+        if(actionBar != null) {
+            actionBar.setDisplayHomeAsUpEnabled(true)
+            actionBar.title = "Create Activity"
+        }
+    }
+    // This is the function that's called when the back button on the action bar is pressed
+    override fun onOptionsItemSelected(item: MenuItem): Boolean {
```

Comment:
Run code formatter with the rules that I posted on discord as the semicolons do not follow the style